spec/factories/questions.rb

```diff
@@ -3,7 +3,7 @@
     title { Faker::StarWars.quote }
     content { Faker::Hipster.paragraph }
     views { rand(5) }
-    user
+    user factory: :user
```

Comment:
I see @andela-oakinniranye. Probably something else was causing my problem then. I have removed it and it works fine.